### ISSUE TITLE
[a11y]: Uploads view

### ISSUE THREAD

**Issue by lsr-explore** [usdigitalresponse/usdr-gost#3837]
## Issue 1 - validation announcement - This is also an axe DevTools error
Voiceover does not announce the "Validated on 12:03:32..." description when navigated in order.  It does announce it when navigated backwards.
https://github.com/user-attachments/assets/0e1e9b41-c7e6-4585-bcf5-5b11b7ba36ab

![image](https://github.com/user-attachments/assets/974923db-6ef3-4179-8396-78133ad7035f)

![image](https://github.com/user-attachments/assets/107fc821-402a-4b6b-8e3e-3fee2e869e48)

Proposed solution - needs to be verified
- Mark the svg as decorative
  - role="img"
  - aria-hidden="true"

----
# Issue 2 - Improve the navigation of the selected upload record

The eighth item is not announced as it is the button group.  
https://github.com/user-attachments/assets/a321521e-2fa6-48b4-b74b-1a9c488d3b10

![image](https://github.com/user-attachments/assets/2e72d116-e853-47e2-8ef6-856bc58f1b97)

Proposed solution
- Refactor so that it isn't a unordered list
- OR remove the buttons from the list.

----
# Issue 3 - When selecting an item from the table - need an indication that it has been loaded.  Focus should be set to the Upload title

----
# Issue 4 -  axe DevTools error - Heading levels should only increase by one
![image](https://github.com/user-attachments/assets/f5bd2fa7-d101-499e-a264-a17713f24833)

----

### Current State

When user visits X page...

### Expected State

When user visits X page...

### Implementation Plan

The following functions need to be added...
